internal/security/config/command/proxy/adduser/command.go

```diff
@@ -66,19 +69,20 @@ func NewCommand(
 	flagSet.StringVar(&dummy, "confdir", "", "") // handled by bootstrap; duplicated here to prevent arg parsing errors
 	flagSet.StringVar(&cmd.tokenType, "token-type", "", "Type of token to create: jwt or oauth2")
 	flagSet.StringVar(&cmd.username, "user", "", "Username of the user to add")
-	flagSet.StringVar(&cmd.group, "group", "admin", "Group to which the user belongs, defaults to 'admin'")
+	flagSet.StringVar(&cmd.group, "group", "gateway", "Group to which the user belongs, defaults to 'gateway'")
 
 	flagSet.StringVar(&cmd.algorithm, "algorithm", "", "Algorithm used for signing the JWT, RS256 or ES256")
 	flagSet.StringVar(&cmd.publicKeyPath, "public_key", "", "Public key (in PEM format) used to validate the JWT.")
 	flagSet.StringVar(&cmd.jwtID, "id", "", "ID to use for linkage with JWT claim (usually the 'iss' field)")
+	flagSet.StringVar(&cmd.jwtFile, "jwt", "", "The file containing the JWT for use when accessing the Kong Admin API")
```

Comment:
should we treat this as raw jwt string or is it a filename?   This affects how you test it.

Reply:
Additionally, it looks a lot like it overlaps with the previous.  Maybe "adminjwtfile" or something that makes to clear which JWT we are talking about.